content/en/docs/tasks/access-kubernetes-api/configure-aggregation-layer.md

```diff
@@ -224,6 +224,55 @@ If you are not running kube-proxy on a host running the API server, then you mus
 
 {{% /capture %}}
 
+### Register APIService objects
```

Comment:
given this section is about creating an APIService, potentially this fits under Step 14 in https://kubernetes.io/docs/tasks/access-kubernetes-api/setup-extension-api-server/

```
14. Create a Kubernetes apiservice. The CA cert above should be base64 encoded, stripped of new lines and used as the spec.caBundle in the apiservice. This should not be namespaced. If using the kube-aggregator API, only pass in the PEM encoded CA bundle because the base 64 encoding is done for you.
```

but to be honest that doc is poorly documented.. I don't feel strongly whether moving this section over will make it read better. Happy to lgtm if you think the current place is good enough

Reply:
Thanks for looking around @roycaihw. I had seen this section as well. I wasn't sure the ordered list wasn't a great place to add more detail. Let's keep what we have in the PR now.